### DISCUSSION TITLE
Data sample for matting

### DISCUSSION THREAD

**Discussion by AceFromSpace** [open-mmlab/mmediting#1212] [Datasets]
Hi

Can someone provide some dataset sample for matting models? Right now the only way to obtain that is to run script preprocess_comp1k_dataset.py on Composition-1k dataset. But I don't have a access to this dataset (They told me access is only for universities). So can anyone provide some sample of his  custom data in proper format? 

To be clear,  I don't want Composition-1k data just some custom sample!

**Comment by Uxell** [comment]
Hi @AceFromSpace,
I stumbled upon a similar issue. Any sample or even just a template would be very helpful!

**Reply by tomasz-gawron-wttech** [reply]
Hello, I've been trying to prepare custom dataset so that I can retrain the matting model and got stuck. Such a sample dataset would help a ton!
